control.yml

```diff
@@ -1,3 +1,4 @@
 - import_playbook: "{{ playbook_dir }}/deploy-with-{{ provider | default('lago') }}.yml"
 - import_playbook: "{{ playbook_dir }}/deploy-{{ cluster_type | default('openshift') }}.yml"
-- import_playbook: "{{ playbook_dir }}/install-kubevirt-release.yml"
+- import_playbook: "{{ playbook_dir }}/playbooks/login-to-lago.yml"
```

Comment:
maybe this was discussed already and i missed it, but i wonder why are we making lago part of this deployment?
i would imagine that if somebody wants to use lago, he would have his own playbook with needed params/roles importing kubevirt-ansible and not forcing it here...

Reply:
i think it should be renamed.
role/playbook name should indicate what it does and not have some unrelated name